frontend/src/features/notes/detail/NoteDetailsFormModal.tsx

```diff
@@ -79,6 +79,7 @@ export const NoteDetailsFormModal: React.FC<INoteDetailsFormModalProps> = props
       message={loading ? spinner : body}
       okButtonText="Close"
       handleOk={() => onCloseClick && onCloseClick()}
+      handleCancel={() => onCloseClick && onCloseClick()}
```

Comment:
these could be avoided if the setDisplay param is passed. Also, given the function have the same parameters it could be passed directly 
```
handleOk={onCloseClick}
```